internal/config/config_test.go

```diff
@@ -134,16 +135,16 @@ version: "2"`
 repo: github.com/example/project
 version: "3-alpha"
 plugins:
-  plugin-x:
```

Comment:
I don't think these indentation changes are needed.

Reply:
That was required to make the tests pass. Otherwise, it fails since we are checking if string == string.

Reply:
This case does not check string equality. It passes locally for me without these indent changes.